nest_elephant_tvb/translation/mpi_translator.py

```diff
@@ -6,12 +6,46 @@
 
 lock_status=Lock() # locker for manage the transfer of data from thread
 
-def init(path, param, comm_receiver, comm_sender, logger_master, logger_receive, logger_send):
+def init(path, param, comm, comm_receiver, comm_sender, loggers):
+    '''
+    Initialize the translation with MPI
+    
+    '''
     
+    # destructure logger list to indivual variables
+    logger_master, logger_receive, logger_send = loggers
+    # science part, see import
     store = store_data(path+'/log/',param)
```

Comment:
If you are busy why not use os.path (or some other lib) to make this file generation a little less fragile

Reply:
Or add a todo

Reply:
```suggestion
    # TODO: use os.path (or similar) for proper file handling.
    store = store_data(path+'/log/',param)
```